### ISSUE TITLE
Old elo's pattern

### ISSUE THREAD

**Issue by maxforasteiro** [jessepollak/payment#89]
Hey there, didn't find any guideline to open a issue in this repo, neither to open a pull request, but here i am haha!

So, my issue is with Elo's credit card number validation, some new numbers aren't being detected.

The fix is simple, just update the actual regex to the new one, based on this gist comment https://gist.github.com/erikhenrique/5931368#gistcomment-1683630

```diff --git a/src/index.coffee b/src/index.coffee
index bbc9246..a384543 100644
--- a/src/index.coffee
+++ b/src/index.coffee
@@ -93,7 +93,7 @@ cards = [
   }
   {
       type: 'elo'
-      pattern: /^(4011(78|79)|43(1274|8935)|45(1416|7393|763(1|2))|50(4175|6699|67[0-7][0-9]|9000)|627780|63(6297|6368)|650(03([^4])|04([0-9])|05(0|1)|4(0[5-9]|3[0-9]|8[5-9]|9[0-9])|5([0-2][0-9]|3[0-8])|9([2-6][0-9]|7[0-8])|541|700|720|901)|651652|655000|655021)/,
+      pattern: /^401178|^401179|^431274|^438935|^451416|^457393|^457631|^457632|^504175|^627780|^636297|^636369|^(506699|5067[0-6]\d|50677[0-8])|^(50900\d|5090[1-9]\d|509[1-9]\d{2})|^65003[1-3]|^(65003[5-9]|65004\d|65005[0-1])|^(65040[5-9]|6504[1-3]\d)|^(65048[5-9]|65049\d|6505[0-2]\d|65053[0-8])|^(65054[1-9]|6505[5-8]\d|65059[0-8])|^(65070\d|65071[0-8])|^65072[0-7]|^(65090[1-9]|65091\d|650920)|^(65165[2-9]|6516[6-7]\d)|^(65500\d|65501\d)|^(65502[1-9]|6550[3-4]\d|65505[0-8])/,
       format: defaultFormat
       length: [16]
       cvcLength: [3]
```

Thanks!

**Comment by lucascavalheiro** [comment]
Hello there, I think this regex is already outdated :)
Here is a regex including `636368` and `650921 to 650978`:
```/^401178|^401179|^431274|^438935|^451416|^457393|^457631|^457632|^504175|^627780|^636297|^636369|^636368|^(506699|5067[0-6]\d|50677[0-8])|^(50900\d|5090[1-9]\d|509[1-9]\d{2})|^65003[1-3]|^(65003[5-9]|65004\d|65005[0-1])|^(65040[5-9]|6504[1-3]\d)|^(65048[5-9]|65049\d|6505[0-2]\d|65053[0-8])|^(65054[1-9]|6505[5-8]\d|65059[0-8])|^(65070\d|65071[0-8])|^65072[0-7]|^(65090[1-9]|65091\d|650920)|^(65165[2-9]|6516[6-7]\d)|^(65500\d|65501\d)|^(65502[1-9]|6550[3-4]\d|65505[0-8])|^(65092[1-9]|65097[0-8])/```

**Comment by gilbarbara** [comment]
ELO Sucks! :P

**Comment by melloware** [comment]
This was released in 2.4.0
